modules/ROOT/pages/apply-configure-json-threat-task.adoc

```diff
@@ -9,30 +9,38 @@ endif::[]
 >s|Policy Name|JSON Threat Protection
 >s|Summary|Protects against malicious JSON in API requests
 >s|Category|Security
->s|Since Mule Version|3.8.0
+>s|Available from Mule version|3.8.0
 >s|Returned Status Codes|400 - Bad Request
 |===
 
-== Purpose
+Applications processing JSON requests are susceptible to attacks characterized by unusual inflation of elements and nesting levels. Attackers use recursive techniques to consume memory resources. Dramatic swings in the size of the application data often signal a security problem. The JSON Threat Protection policy helps protect your applications from such intrusions.
 
-JSON requests are susceptible to attacks characterized by unusual inflation of elements and nesting levels. Attackers use recursive techniques to consume memory resources. Dramatic swings in the size of the application data often signal a security problem. The JSON threat protection policy helps protect your applications from such intrusions.
+If any possible attacks to your Anypoint Platform setup go undetected, consider designing your services architecture with layers of protection in addition to these policies.
 
-In the event Mule Runtime fails to detect an attack, you need to monitor and design your services architecture with layers of protection in addition to these policies.
+== How This Policy Works
```

Comment:
WHICH policy? Is this about JSON or about "layers of protection in addition to"?
Name the policy in the title, so that someone scanning will know to what "this" refers.

Reply:
Removing this section as there is nothing to document in that part.